packages/lab/src/progress/CircularProgress.css

```diff
@@ -109,3 +106,13 @@
     stroke-dashoffset: -125px;
   }
 }
+
+/* TK1 Backwards compatibility */
+
+.backwardsCompat.uitkCircularProgress .uitkCircularProgress-gradientStart {
+  stop-color: var(--backwardsCompat-measure-fill-startColor);
+}
+
+.backwardsCompat.uitkCircularProgress .uitkCircularProgress-gradientStop {
+  stop-color: var(--backwardsCompat-measure-fill-stopColor);
```

Comment:
for consistency, should it be `measured`?

Reply:
good spot, fixed